keyboards/annepro2/annepro2.c

```diff
@@ -76,30 +73,34 @@ void OVERRIDE keyboard_post_init_kb(void) {
     wait_ms(15);
 
     // loop to clear out receive buffer from ble wakeup
-    while(!sdGetWouldBlock(&SD1))
-        sdGet(&SD1);
+    while (!sdGetWouldBlock(&SD1)) sdGet(&SD1);
+
+    annepro2LedGetStatus();
 
     keyboard_post_init_user();
 }
 
-void OVERRIDE matrix_init_kb(void) {
-    matrix_init_user();
-}
+void OVERRIDE matrix_init_kb(void) { matrix_init_user(); }
 
 void matrix_scan_kb() {
     // if there's stuff on the ble serial buffer
     // read it into the capslock struct
-    while(!sdGetWouldBlock(&SD1)) {
-        sdReadTimeout(&SD1, (uint8_t *) &BLECapsLock, sizeof(ble_capslock_t), 10);
+    while (!sdGetWouldBlock(&SD1)) {
+        sdReadTimeout(&SD1, (uint8_t *)&BLECapsLock, sizeof(ble_capslock_t), 10);
 
         // if it's capslock from ble, darken led
         if (BLECapsLock.caps_lock) {
-            annepro2LedClearMask(MATRIX_COLS * 2);
+            // annepro2LedClearMask(MATRIX_COLS * 2);
```

Comment:
I haven't done anything with it. Shouldn't that set "capslock" in led_t and be processed by user function? Maybe set this and call it instead of a direct clear/set?